libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -317,6 +317,12 @@ bool AP_Scripting::arming_checks(size_t buflen, char *buffer) const
     return true;
 }
 
+void AP_Scripting::restart()
+{
+    _stop = true;
+    _restart = true;
```

Comment:
possible race? may need to set _restart first

Reply:
done